packages/website/components/page-templates/package-home.js

```diff
@@ -1,6 +1,8 @@
 import * as PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { colors } from '@atlaskit/theme';
+import WarningIcon from '@atlaskit/icon/glyph/warning';
+import Banner from '@atlaskit/banner';
```

Comment:
Probably better to use [section message](https://atlaskit.atlassian.com/packages/core/section-message) for this, as banner should only be shown at the top of a page.